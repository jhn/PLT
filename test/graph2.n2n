fn graph2() -> Void {
	actor: Node = { name: String, age: Int }
	actedIn: Rel = { role: String }
	movie: Node = { title: String, year: Int }

	Cast: Graph = (
		actor["Keanu", 35] actedIn["Neo"] movie["Matrix", 1999],
		actor["Leo", 20] actedIn["Jordan"] movie["Wolf", 1994]
	)

	print(Cast)

}