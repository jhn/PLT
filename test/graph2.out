< {Keanu, 35} {Neo} {Matrix},
  {Leo, 20} {Jordan} {Wolf}
>